server/routes/shared/showReferralPresenter.ts

```diff
@@ -75,6 +77,48 @@ export default class ShowReferralPresenter {
         ]
   }
 
+  get responsibleOfficersDetails(): SummaryListItem[][] {
+    return this.responsibleOfficers.map(responsibleOfficer => {
+      const { staff } = responsibleOfficer
+
+      if (!staff) {
+        // This is a temporary step - I want to not break the page if fields are missing, most likely with dev data.
```

Comment:
this looks great - do you think it's ok though that the page will not load if there are no ROs returned for the SU in delius?

Reply:
Yeah I've stopped raising an error now and will just render a "Not found" section here as we have too many places where the journey can break so far.

Reply:
sounds good to me - thanks